src/main/java/org/schedulesdirect/api/Station.java

```diff
@@ -80,13 +80,13 @@ public void writeImageToFile(File dest) throws IOException {
 		/**
 		 * @return the url
 		 */
-		URL getUrl() {
+		public URL getUrl() {
```

Comment:
Yes, there is some PVR software that uses the URL for images (Windows Media Center for example).  We don't need to force the user to download each image to disk first to get a local file URL.